### PR TITLE
chore: remove empty styles and make className last in up-comps

### PR DESCRIPTION
**Pull request by schakraborty22**
### Summary:
This is PR to implement the removal of empty styles classNames in the upcoming components and make the className props last in the conditional styling (cslx) so that the dev styles will override the props styles passed to the components:
Story  id -
204928 : https://app.shortcut.com/czi-edu/story/204928/make-classname-prop-the-last-class-in-list-for-all-components
211218: https://app.shortcut.com/czi-edu/story/211218/remove-classnames-that-don-t-have-corresponding-styles

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,7 +88,8 @@
     "react-focus-lock": "^2.9.1",
     "react-portal": "^4.2.1",
     "react-uid": "^2.3.1",
-    "svg4everybody": "^2.1.9"
+    "svg4everybody": "^2.1.9",
+    "yarn": "^1.22.19"
```
Comment (jinlee93) [inline package.json:92]: These changes shouldn't have any package changes. 
Yarn should be installed globally with `npm install --global yarn`
Can you remove this line, delete `package-lock.json` and revert changes to `yarn.lock`?

**Comment by jinlee93** [comment]
Could you revert the changes to `yarn.lock` as well?

File: src/components/FieldsetItems/FieldsetItems.tsx [hunk 2]
```diff
@@ -31,6 +31,5 @@ export const FieldsetItems = ({
   className,
 }: FieldsetItemsProps) => {
   const componentClassName = clsx(className, styles['fieldset-items']);
-  // @ts-expect-error TODO: investigate error
```
Comment (jinlee93) [inline src/components/FieldsetItems/FieldsetItems.tsx:34]: Why was this TODO removed?

File: src/upcoming-components/AccordionPanel/AccordionPanel.tsx [hunk 3]
```diff
@@ -142,10 +142,7 @@ export const AccordionPanel = ({
         id={ariaControlsVar}
         style={{ height: heightVar }}
       >
-        <div
-          className={styles['accordion__panel-body-inner']}
-          data-accordion-panel
-        >
+        <div className={className} data-accordion-panel>
```
Comment (jinlee93) [inline src/upcoming-components/AccordionPanel/AccordionPanel.tsx:145]: This should not be changed

**Comment by jinlee93** [comment]
There are a lot of instances where `className` has been changed in the returning ReactComponent, such as `className={styles['accordion__panel-body-inner']}` to `className={className}`, `className={styles['checkbox-field__label']}` to `className={className}` need to revert those if they refer to something in the stylesheet or just remove that prop if there is no reference

File: src/upcoming-components/AccordionPanel/AccordionPanel.tsx (hunk 3 above)
Comment (jinlee93) [reply src/upcoming-components/AccordionPanel/AccordionPanel.tsx:145]: Sorry, the `className` prop itself should be removed if no reference in stylesheet

File: package.json (hunk 1 above)
Comment (schakraborty22) [reply package.json]: Thanks ,reverted the package.json and changes to yarn.lock

Comment (jinlee93) [reply package.json]: I think I'm still seeing changes to `yarn.lock` 
image

File: src/components/FieldsetItems/FieldsetItems.tsx (hunk 2 above)
Comment (schakraborty22) [reply src/components/FieldsetItems/FieldsetItems.tsx]: Thanks , added the comment line back

File: src/upcoming-components/CheckboxField/CheckboxField.tsx [hunk 4]
```diff
@@ -93,18 +93,18 @@ export const CheckboxField = ({
...
     isError && styles['eds-is-error'],
     disabled && styles['eds-is-disabled'],
+    className,
   );
   return (
     <Fieldset className={componentClassName} id={id} {...other}>
       <FieldsetLegend
         aria-describedby={fieldNote && ariaDescribedByVar}
-        className={styles['checkbox-field__label']}
+        className={className}
```
Comment (jinlee93) [inline src/upcoming-components/CheckboxField/CheckboxField.tsx:107]: Think this needs to be deleted

File: src/upcoming-components/MediaBlock/MediaBlock.tsx [hunk 5]
```diff
@@ -44,18 +44,14 @@ export const MediaBlock = ({
...
   return (
     <div className={componentClassName} {...other}>
       <div className={styles['media-block__body']}>{children}</div>
       <div className={styles['media-block__media']}>
-        <img
-          alt={imgAlt}
-          className={styles['media-block__image']}
-          src={imgSrc}
-        />
+        <img alt={imgAlt} className={styles['media-block__img']} src={imgSrc} />
```
Comment (jinlee93) [inline src/upcoming-components/MediaBlock/MediaBlock.tsx:54]: Good catch

**Review by jinlee93 — APPROVED** [review]
Thanks for the changes, LGTM

**Pull request merged**